src/test/java/edu/unc/lib/boxc/migration/cdm/services/GroupMappingServiceTest.java

```diff
@@ -105,18 +101,16 @@ public void generateSingleRunTest() throws Exception {
         service.generateMapping(options);
 
         GroupMappingInfo info = service.loadMappings();
-        String group1Key = info.getGroupKeyByMatchedValue("groupa:group1");
-        String group2Key = info.getGroupKeyByMatchedValue("groupa:group2");
-        assertMappingPresent(info, "25", "groupa:group1", group1Key);
-        assertMappingPresent(info, "26", "groupa:group1", group1Key);
-        assertMappingPresent(info, "27", "groupa:group2", group2Key);
-        assertMappingPresent(info, "28", null, null);
-        assertMappingPresent(info, "29", null, null);
+        assertMappingPresent(info, "25", "groupa:group1");
```

Comment:
both this first set of `assertMappingPresent` assertions match the set of assertions on line 152, 173, and 190. they're supposed to be the same from all the different runs, yes? might be nice to DRY that up?

Reply:
added a common assertion method